core/include/kernel/tee_misc.h

```diff
@@ -49,4 +49,23 @@ bool core_is_buffer_outside(paddr_t b, paddr_size_t bl,
 bool core_is_buffer_intersect(paddr_t b, paddr_size_t bl,
 			      paddr_t a, paddr_size_t al);
 
+/**
+ * Allocate outer cache line aligned memory buffer.
```

Comment:
Please update the description to make it clear that it's the max cache line size.

Reply:
OK